src/content/docs/tutorial-errors/reduce-common-errors.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Reduce common errors
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+So far, you've learned how to scope out a service outage and get the error in front of the correct teams. Error analysis doesn't end after the solution is deployed, however. To return to our flooded yard metaphor, you've only diagnosed the leaking pipe as the direct cause of the flood, then ordered a patch to fix it. You aren't done, though. At this point, you should ask yourself: What caused the leaking pipe to begin with? 
+
+## Objectives [#objectives]
+
+This tutorial helps you understand the difference between direct cause and root cause so you can reduce common errors over time. By the end of this tutorial, you will be able to:
+
+* Route errors to the correct teams
```

Comment:
a little tenuous how these two are related. can these be tied together more closely?